UE4SS/src/GUI/LiveView.cpp

```diff
@@ -279,6 +280,137 @@ namespace RC::GUI
     };
     FLiveViewDeleteListener FLiveViewDeleteListener::LiveViewDeleteListener{};
 
+    static auto add_bool_filter_to_json(JSON::Array& json_filters, const StringType& filter_name, bool is_enabled) -> void {
+        auto json_object = std::make_unique<JSON::Object>();
+        json_object->new_string(STR("FilterName"), filter_name);
+        auto& filter_data = json_object->new_object(STR("FilterData"));
+        filter_data.new_bool(STR("Enabled"), is_enabled);
+        json_filters.add_object(std::move(json_object));
+    }
+
+    template <typename ContainerType>
+    static auto add_array_filter_to_json(JSON::Array& json_filters, const StringType& filter_name, const ContainerType& container, const StringType& array_name) -> void {
+        auto json_object = std::make_unique<JSON::Object>();
+        json_object->new_string(STR("FilterName"), filter_name);
+        auto& filter_data = json_object->new_object(STR("FilterData"));
+
+        if (array_name == STR("ClassNames"))
+        {
+            filter_data.new_bool(STR("IsExclude"), Filter::ClassNamesFilter::b_is_exclude);
+        }
+
+        auto& values_array = filter_data.new_array(array_name);
+        for (const auto& value : container) {
+            if constexpr (std::is_same_v<typename ContainerType::value_type, FName>) {
+                StringType string_value = value.ToString();
+                values_array.new_string(string_value);
```

Comment:
```suggestion
                values_array.new_string(value.ToString());
```
I think this is an unnecessary temporary variable ?
It doesn't generate a compiler error.

Reply:
Just didn't think about embedding it into the same line when adding it.